homeassistant/components/camera/__init__.py

```diff
@@ -413,5 +425,16 @@ class CameraMjpegStream(CameraView):
 
     @asyncio.coroutine
     def handle(self, request, camera):
-        """Serve camera image."""
-        yield from camera.handle_async_mjpeg_stream(request)
+        """Serve camera stream, possibly with interval."""
+        if request.query.get('interval'):
+            """Compose camera stream from stills."""
+            interval = float(request.query.get('interval'))
+            if interval < MIN_STREAM_INTERVAL:
+                return web.Response(status=403,
```

Comment:
This is not a 403, it's a bad request (400)

Reply:
You are probably right, I think I just misunderstood the 403 description.